### PR TITLE
feat: Add Nim to scope languages

### PR DESCRIPTION
**Pull request by janAkali**
https://github.com/nvim-treesitter/nvim-treesitter/blob/eff88f4e8b71d92bdcaeca6e4f7e1e146b263614/queries/nim/locals.scm#L291-L322

### CODE REVIEW

**Comment by lukas-reineke** [comment]
Thank you, I think everything looks good. 
Please give me a couple of days to find some time to test this.

File: lua/ibl/scope_languages.lua [hunk 1]
```diff
@@ -298,6 +298,27 @@ local M = {
     mlir = {
         region = true,
     },
+    nim = {
+        ["if"] = true,
+        case = true,
+        try =  true,
```
Comment (lukas-reineke) [inline lua/ibl/scope_languages.lua:304]: Please remove the extra space to make CI happy

```suggestion
        try = true,
```

**Review by lukas-reineke — CHANGES_REQUESTED** [review]

File: lua/ibl/scope_languages.lua (hunk 1 above)
Comment (janAkali) [reply lua/ibl/scope_languages.lua]: Done.

**Review by lukas-reineke — APPROVED** [review]
thanks

**Pull request merged**